migrations/20240501024340_vnft_into_ud_tbl.sql

```diff
@@ -0,0 +1,109 @@
+-- +goose Up
+-- +goose StatementBegin
+SET search_path = devices_api, public;
+
+ALTER TABLE user_devices
+    ADD COLUMN mint_request_id char(27) CONSTRAINT user_devices_mint_request_id_fkey UNIQUE REFERENCES meta_transaction_requests(id);
+
+ALTER TABLE user_devices
+    ADD COLUMN burn_request_id char(27) CONSTRAINT user_devices_burn_request_id_fkey UNIQUE REFERENCES meta_transaction_requests(id);
+
+ALTER TABLE user_devices
+    ADD COLUMN token_id numeric(78, 0) CONSTRAINT user_devices_token_id_key UNIQUE;
+
+ALTER TABLE user_devices
+    ADD COLUMN claim_id varchar CONSTRAINT user_devices_claim_id_fkey UNIQUE REFERENCES verifiable_credentials(claim_id);
+
+ALTER TABLE user_devices
+    ADD COLUMN owner_address BYTEA CONSTRAINT user_devices_owner_address_check CHECK (length(owner_address) = 20);
+
+UPDATE user_devices 
+    SET 
+        burn_request_id = vnft.burn_request_id,
+        mint_request_id = vnft.mint_request_id,
+        token_id = vnft.token_id,
+        claim_id = vnft.claim_id,
+        owner_address = vnft.owner_address
+FROM vehicle_nfts vnft 
+WHERE vnft.user_device_id = user_devices.id;
```

Comment:
Note that we still have some rows in `vehicle_nfts` with null `user_device_id`. We should look into those.